src/lib/editor/Editor.svelte

```diff
@@ -195,8 +196,15 @@
       monaco.editor.remeasureFonts();
     });
 
-    envelope.subscribe((value) => {
-      monacoEditor.updateOptions({ readOnly: Boolean(value?.sigs) });
+    envelope.subscribe(async (value) => {
+      const isSigned = Boolean(value?.sigs);
+      if (!isSigned) {
+        monacoEditor.updateOptions({ readOnly: false });
+        return;
+      }
+
+      await tick();
```

Comment:
Nice! Given that it's not obvious what `tick()` means, I'd strongly suggest adding some comments around this to explain the situation!

Reply:
Agreed!